src/types/frame.ts

```diff
@@ -132,7 +132,7 @@ export type FrameResponseFn = (
 ) => TypedResponse<FrameResponse>
 
 export type FrameData = {
-  address?: string | undefined
+  address?: Address | string | undefined
```

Comment:
```suggestion
  address?: Address | undefined
```

Reply:
I suppose that would break some types, so there should be type casting used somewhere at the `address` retrieval.

Reply:
I would prefer to keep the string in the type. We specified it as a string to future-proof for non-EVM chains.